frontend/src/DiagramEditor/DiagramEditor.jsx

```diff
@@ -129,28 +134,38 @@ export default class DiagramEditor extends React.Component {
     });
   };
 
-  onMouseWheel = (e) => {
+  zoom = (e) => {
```

Comment:
I'm guessing that you're not done with this fuction as there are several commented lines and a console log...

Reply:
it's not quite done, no. But I'd rather leave that code in for now, until I can get around to finishing it